stable/rabbitmq-ha/templates/ingress.yaml

```diff
@@ -28,10 +28,13 @@ spec:
               servicePort: {{ .Values.rabbitmqManagerPort }}
 {{- if .Values.ingress.tls }}
   tls:
-    - secretName: {{  .Values.ingress.tlsSecret }}
       {{- if .Values.ingress.hostName }}
       hosts:
         - {{ .Values.ingress.hostName }}
       {{- end }}
+      secretName: {{  .Values.ingress.tlsSecret }}
+      {{- else}}
+      - secretName: {{  .Values.ingress.tlsSecret }}
```

Comment:
I don't think there's anything wrong with the way it was before, but I know there's something wrong with the way you've changed it. Now `secretName` is sometimes a list and sometimes not a list and the tls is no longer an array. Please see the ingress [spec](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#ingressspec-v1beta1-networking-k8s-io). You also haven't signed the DCO or bumped the chart version